json_templates.rdl_json_exposure1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "properties": {
```

Comment:
The `properties` key is part of the structure of the JSON schema but does not appear in JSON data so it should be omitted.